README.md

```diff
@@ -57,18 +57,20 @@ Monero. See [here](https://github.com/fireice-uk/cryptonote-speedup-demo/) for d
 
 If your prefered coin is not listed, you can choose one of the following algorithms:
 - 256Kib scratchpad memory
-    - cryptonight_turtle
+    - cryptonight_turtle (equal to cryptonight_v8 but `0x1FFF0` as mask, 1/8 of iterations and scratchpad size)
 - 1MiB scratchpad memory
     - cryptonight_lite
     - cryptonight_lite_v7
     - cryptonight_lite_v7_xor (algorithm used by ipbc)
 - 2MiB scratchpad memory
     - cryptonight
+    - cryptonight_gpu (for Ryo 02/2019 fork)
```

Comment:
Let's make it `14th of Feb fork` - remember they write the dates opposite way on the other side of the pond.

Reply:
I prefer `YYYY-MM(-DD)` because everywhere across every pond does it wrong.
Significance ordering is always the right order, then it sorts properly as text also, and easy to read and comprehend/compare quickly without lexical errors.

Reply:
Exactly the reason why I prefer letters -> no matter the order "14th of Feb" "Feb 14th", the meaning is the same to everyone.